relay.js

```diff
@@ -1,4 +1,5 @@
 import {verifySignature} from './event'
+const WS = typeof window === 'undefined' ? require('ws') : WebSocket
```

Comment:
Does this require work when building for the browser? I think any transpiler will always automatically include the `ws` package since it will be available.

Reply:
there is no build for the browser, see `package.json`

```
  "main": "dist/nostr-tools.cjs.js",
  "module": "index.js",
  "browser": "index.js",
```

Reply:
Well, I guess there is a build for anyone using the library, so their build would be bulky, don't know if there is a better way.

Reply:
I added a separate `index.cjs.js` file which imports and exports the original, leaving `relay.js` untouched, this should eliminate the `ws` in the browser when building by third parties using `nostr-tools`